apps/enc.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 1995-2017 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2017 BaishanCloud. All Rights Reserved.
```

Comment:
Please remove this line. We generally don't add an extra copyright line like this unless there is a major contribution (e.g. usually when the file is created)